packages/devtools_app/lib/src/vm_flags.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+const asyncDebugger = 'async_debugger';
```

Comment:
add a comment perhaps linking to where the flags are defined in the VM in the dartsdk repo.

Reply:
done.